README.md

```diff
@@ -57,40 +57,52 @@ actions: {
 
 ```js
 actions: {
-    saveOptions() {
-        this.get('model').save()
-        .then(() => {
-            this.notifications.clearAll();
-            this.notifications.addNotification({
-                message: 'Successfully saved your settings',
-                type: 'success'
-            });
-        })
-    }
+  saveOptions() {
+    this.get('model').save()
+    .then(() => {
+      this.notifications.clearAll();
+      this.notifications.addNotification({
+        message: 'Successfully saved your settings',
+        type: 'success'
+      });
+    })
+  }
 }
 ```
+
+### Set a global, default duration time
+
+This code only needs to be called in one place such as your application route.
+
+```js
+this.notifications.setDefaultClearNotification(1000);
+```
+
 ### Template
 
 Include this snippet in your Handlebars template to display the notifications.
 
 ```hbs
 <div class="c-notification__container">
-    {{#each notifications as |notification|}}
-        {{notification-message notification=notification}}
-    {{/each}}
+  {{#each notifications as |notification|}}
+    {{notification-message notification=notification}}
+  {{/each}}
 </div>
 ```
 
-## Font Awesome
+## Icons
 
-[Font Awesome] is required as part of the addon to display the message type icons on the notifications.
+By default, [Font Awesome] is used for the message type icons on the notifications. Alternatively, you can use [Glyphicons] that are packaged with Bootstrap.
 
-If Font Awesome is not already included in the consuming application, add the following to your applications `config/environment.js` file as a property of the `ENV` object.
+If Font Awesome is not already included in the consuming application, the addon can add it via Bower. Glyphicons can **not** be added to your application via this addon.
+
+Add the following to your applications `config/environment.js` file as a property of the `ENV` object.
 
 ```js
 var ENV = {
   'ember-cli-notifications': {
-    includeFontAwesome: true
+    icons: 'font-awesome', // Or 'bootstrap'
```

Comment:
this constitutes a breaking change, will need to do a major bump for this. maybe we could come up with a different API that wouldn't break backwards compatibility, probably worth a chat